exercises/practice/automated-readability-index/.docs/instructions.md

```diff
@@ -0,0 +1,110 @@
+# Instructions
+
+As we grow up, we can understand more and more complex texts. 
+But how is the level of complexity of a given text assessed, 
+and how can we make programs which do that? In this exercise, 
```

Comment:
[Exercism's style guide for markdown](https://exercism.org/docs/building/markdown/markdown) uses one sentence per line. I resisted at first but have come to see the wisdom behind this style.

Reply:
fixed